hack/run-dptp-controller-manager.sh

```diff
@@ -27,15 +27,8 @@ kubectl --context build01 get secret -n ci regcred  -o json  \
 go build  -v -o /tmp/dptp-cm ./cmd/dptp-controller-manager
 /tmp/dptp-cm \
   --leader-election-namespace=ci \
-  --ci-operator-config-path="$(go env GOPATH)/src/github.com/openshift/release/ci-operator/config" \
-  --config-path="$(go env GOPATH)/src/github.com/openshift/release/core-services/prow/02_config/_config.yaml" \
-  --job-config-path="$(go env GOPATH)/src/github.com/openshift/release/ci-operator/jobs" \
```

Comment:
Can you leave the config path variables? They are not needed now, but make it easier when someone wants to test one of the controllers that do need them

Reply:
This wont be merged for real: Just want you to check the script that I plan to run.
I will recovery this before merge.

Reply:
Have to recover them because they are mandatory flags.